### ISSUE TITLE
ordered-imports rule doesn't sort in the same order as Organize Imports in TypeScript

### ISSUE THREAD

**Issue by aboyton** [palantir/tslint#4063]
### Bug Report

- __TSLint version__: master
- __TypeScript version__: 2.9
- __Running TSLint via__: CLI

See https://github.com/Microsoft/TypeScript/issues/25114

#### TypeScript code being linted

```ts
import { foo, _bar } from 'baz';
```

with `tslint.json` configuration:

```json
{
  "ordered-imports": true,
}
```

#### Actual behaviour
It sorts it to `import { _bar, foo }`, which is different to how TypeScript organises it.

#### Expected behaviour
No lint error.

The difference seems to be that TypeScript is normalising strings using `toUpperCase` where as `tslint` uses `toLowerCase`.

**Comment by Timebutt** [comment]
Is there a reason this isn't being picked up? This is a feature I (and I'm guessing many others) have been waiting for for a long time.

**Comment by JoshuaKGoldberg** [comment]
@Timebutt it's marked as `Status: Accepting PRs` which means anybody is welcome to send a PR. Including you! 😉

**Comment by Timebutt** [comment]
I know and I'm looking to contribute, was just wondering if there is a specific reason why this wouldn't be fixed? Implementing this will be a breaking change, forcing everybody to change code and this might not be desired. If there is nothing preventing us from fixing this, I will try my hand at implementing this for sure!

**Comment by JoshuaKGoldberg** [comment]
Gotcha, great! 

> just wondering if there might be a specific reason why this wouldn't be fixed
> breaking change

Good call - adding the missing label. We can review a PR and hold off merging it until the next major version, which should be soon.

**Comment by aboyton** [comment]
We have a PR for it, it's #4064. I'd love to see this merged although personally I just forked months ago.

**Comment by Timebutt** [comment]
So we actually have the fix, and it was just waiting for a next major version? In that case, it seems like the timing is in our favour, as a new version will be around soon right @JoshuaKGoldberg?

**Comment by JoshuaKGoldberg** [comment]
Correct! 🙌

**Comment by JoshuaKGoldberg** [comment]
Removing the `Type: Breaking Change` label per #4811. Now accepting PRs!

**Comment by kyro38** [comment]
Is the fix expected to ship in the next release  ?
